### PR TITLE
Update and rename redis-on-ubuntu-14-04.md to deploy-redis-on-ubuntu-…

### PR DESCRIPTION
**Pull request by kcraig34**
…14-04.md

Copy edited. Reviewed and revised for SEO. URL/Title/Description aligned. Keywords modified.

### CODE REVIEW

File: docs/databases/redis/deploy-redis-on-ubuntu-14-04.md [hunk 1]
```diff
@@ -114,20 +115,20 @@ To improve Redis performance, make the following adjustment to the Linux system
...
 
-Starting from version 3.0, there is also a *Redis Cluster*, which is a data sharding solution with automatic management, handling failover and replication. With Redis Cluster you are able to automatically split your dataset among multiple nodes, which is useful when your dataset is larger than a single server's RAM. It also gives you an ability to continue operations when a subset of the nodes are experiencing failures or are unable to communicate with the rest of the cluster.
+Starting with Redis version 3.0, *Redis Cluster*, a data-sharding solution with automatic management, handles failover and replication. With Redis Cluster you are able to automatically split your dataset among multiple nodes, which is useful when your dataset is larger than a single server's RAM. It also gives you the ability to continue operations when a subset of the nodes are experiencing failures or are unable to communicate with the rest of the cluster.
```
Comment (spariev) [inline docs/databases/redis/deploy-redis-on-ubuntu-14-04.md:120]: I would rephrase it as

Starting with Redis version 3.0, you can use _Redis Cluster_ - a data sharding solution, that automatically manages replication and failover.

I think point here is that starting from version 3.0, user can also use the Cluster solution to manage replication/failover, but he/she still can deal with it by other means. And in your variant it sounds to me like from version 3.0, Redis Cluster is the only thing that handles faliover and replication.

Comment (kcraig34) [inline docs/databases/redis/deploy-redis-on-ubuntu-14-04.md:120]: Point is valid. Go ahead and revise as you see fit.
